content/docs/cms/events.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Events
+prev: /docs/cms/apis
+next: /docs/cms/alerts
+---
+
+The **Events** feature allows decoupled parts of the CMS to notify the rest of the system about certain events which have happened.
+
+## Interface
+
+```ts
+interface Events {
+  dispatch(event: Event): void
+  subscribe(event: string, listener: Listener): Unsubscribe
```

Comment:
```suggestion
  subscribe(eventType: string, listener: Listener): Unsubscribe
```

seeing the two `event` params next to each other with different types makes me think it would be helpful to call this `eventType`